src/views/Alteration.vue

```diff
@@ -73,6 +68,12 @@
         />
       </template>
 
+      <staff-payment
+        class="mt-10"
+        :validate="getAppValidate"
```

Comment:
Doesn't look like the validate prop is used anywhere

Reply:
IMHO - leave it (less work). Future ticket (do you have the number?) can update it if needed.

Reply:
from the ticket 

> (to note - the Top Level red bar validation shown on comp 10a2 and auto scroll can be complete in the Top Level validation ticket #https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/6546)